source/connection-monitoring-and-pooling/connection-monitoring-and-pooling.rst

```diff
@@ -1115,6 +1122,23 @@ clear the pool again. This situation is possible if the pool is cleared by the
 background thread after it encounters an error establishing a connection, but
 the ServerDescription for the endpoint was not updated accordingly yet.
 
+Why do we need to request the Background Thread Run in Clear logic?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+If a SDAM logic has observed a network timeout, we assume that all connections
+including "in use" connections are no longer healthy. In some cases connections 
+will fail to detect the network timeout fast enough, for example server request 
+can hang on OS level in TCP retry loop up to 17 minutes until failing. Therefore 
+these connections are proactively closed in the case of SDAM network timeout. 
+This logic is cross driver and not dependent on OS behavior and configured 
+background thread interval.
+
+Why don't we configure TCP_USER_TIMEOUT?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Ideally reasonable TCP_USER_TIMEOUT for connection can help with detecting stale 
+connections as alternative for manual requesting Background Thread Run in Clear. 
+Unfortunately this approach doesn't work on Windows. Also configuring timeouts 
+smaller than 20 seconds proved to be impossible in some cases.
```

Comment:
```suggestion
Ideally, a reasonable TCP_USER_TIMEOUT can help with detecting stale 
connections as an alternative to `closeInUseConnections` in Clear. 
Unfortunately this approach doesn't work on Windows. Also configuring timeouts 
smaller than 20 seconds proved to be impossible in some cases.
```

Reply:
Also, can we provide a bit more context on why this doesn't work on Windows?

Reply:
Also, AFAIK at least java also has no way to configure this behavior via built in options